node_modules/node-red-node-email/61-email.html

```diff
@@ -84,7 +88,8 @@
             dname: {value:""},
             to: {required: true},
             from: {required: true},
-            subject: {required: true}
+            subject: {required: true},
+            body : {required: true}
```

Comment:
Your assumption is right - it would work just like an output device. I'll update this field to look like it.